strawberryfield.routing.yml

```diff
@@ -74,9 +76,10 @@ strawberryfield.flavor_datasource_search:
       resource_type:
         type: 'ado'
       fileuuid: 'all'
+      processor: 'ocr'
+      format: 'json'
+      page: 'all'
   requirements:
-    fileuuid: .+
```

Comment:
WE still need the fileuuid? and the processor?

Reply:
I think we don't need fileuuid anymore while processor (currently only ocr) could be useful for future flavors, right?

Reply:
I think we still need it. Because page itself is not enough when dealing with TIFFs instead of PDFs

Reply:
@DiegoPino I had to change routing yml as you see to make default working  because I want it works also without passing format and page to be backward compatible. Some test give me that, removing those settings from requirements while they are present under parameters. Probably you have more clear idea about this, feel free to make any changes you consider better for code

Reply:
I know this may be more work(one I can do) but maybe we want to have the XML format (since it won't really search right? it is the full thing?) into its own route then. I feel unsure allowing so much unchecked arguments.Give me a few, I have some fire to extinguish on another project. Thanks!!

Reply:
@DiegoPino Full miniocr as stored in Solr is already returned if you use /miniocr/page as in this https://archipelago.byterfly.eu/do/7facaffe-e07d-4931-a5e3-0dc8210d7518/flavorsearch/all/ocr/miniocr/10 is this what you mean? anyway, no rush, friend